adminSiteClient/ImagesIndexPage.tsx

```diff
@@ -88,40 +87,35 @@ function AltTextEditor({
     getAltText: ImageEditorApi["getAltText"]
 }) {
     const [value, setValue] = useState(text)
-    const [shouldAutosize, setShouldAutosize] = useState(false)
 
-    const saveAltText = useCallback(() => {
-        const trimmed = value.trim()
-        patchImage(image, { defaultAlt: trimmed })
-    }, [image, patchImage, value])
+    const saveAltText = useCallback(
+        (newValue: string) => {
+            patchImage(image, { defaultAlt: newValue })
+            setValue(newValue)
+        },
+        [image, patchImage]
+    )
 
     const handleGetAltText = useCallback(async () => {
         const response = await getAltText(image.id)
         setValue(response.altText)
-        // Only autoexpand the textarea if the user generates alt text
-        setShouldAutosize(true)
```

Comment:
Did you want to get rid of this extra logic?

Reply:
nah I actually kinda ended up liking how it emphasizes the rows that have missing alt text 😉